queries/gotmpl/highlights.scm

```diff
@@ -0,0 +1,106 @@
+; Identifiers
+[
+  (field)
+  (field_identifier)
+] @property
+
+(variable) @variable
+
+; Function calls
+(function_call
+  function: (identifier) @function)
+
+(method_call
+  method:
+    (selector_expression
+      field: (field_identifier) @function))
+
+; Builtin functions
+(function_call
+  function: (identifier) @function.builtin
+  (#set! priority 101)
+  (#match? @function.builtin
+    "^(and|call|html|index|slice|js|len|not|or|print|printf|println|urlquery|eq|ne|lt|ge|gt|ge)$"))
```

Comment:
Chang to any-of

Reply:
Another thing is that there is html and js here. So does that mean the arguments for the function can be valid html/js? If yes, more injections!

Reply:
And printf. ;)

Reply:
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/36446499/54fa509f-ab4f-43b3-9d5c-fb219defb1d9)

Do you know why prinf is not really working? In the tree-sitter playground my query was looking good. Printf injection also does not seem to work for bash in my nvim config.

Regarding html and js, I think you could them with almost the same query, but it seems like the quotes are also part of the `interpreted_string_literal` node (see picture). This would require modifying the grammar.

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/36446499/a4162414-7003-488d-b004-11ac5ca70eaf)

Reply:
> Do you know why prinf is not really working?

Did you `:TSInstall printf` ?

> `it seems like the quotes are also part of the interpreted_string_literal`

Use `(#offset! @injection.content 0 1 0 -1)`

Reply:
I installed it.
JS and html worked fine after installing them.